x-pack/packages/kbn-elastic-assistant/impl/assistant/api.tsx

```diff
@@ -11,16 +11,28 @@ import { HttpSetup, IHttpFetchError } from '@kbn/core-http-browser';
 import type { Conversation, Message } from '../assistant_context/types';
 import { API_ERROR } from './translations';
 import { MODEL_GPT_3_5_TURBO } from '../connectorland/models/model_selector/model_selector';
-import { getFormattedMessageContent } from './helpers';
+import {
+  getFormattedMessageContent,
+  getOptionalRequestParams,
+  hasParsableResponse,
+} from './helpers';
 import { PerformEvaluationParams } from './settings/evaluation_settings/use_perform_evaluation';
 
 export interface FetchConnectorExecuteAction {
+  alerts: boolean;
```

Comment:
nit: consider more human readable name for understanding role here. `canQueryAlerts`, `alertQueryingEnabled`?